release-notes.txt

```diff
@@ -1,3 +1,536 @@
+* cpu/cortexm_common: print last active thread on stack corruption (#20709)
+* cpu/esp: Handle format print errors (#20596)
+* cpu/msp430: implement power management (#20613)
+* cpu/nrf5x: implement pm_off() for nRF53/9160 (#20590)
+* cpu/nrf5x_common: fix ztimer issue on warm-boot (#20665)
+* cpu/nrf5x_common: properly calibrate RC-based low-frequency clock (#20669)
+* cpu/sam0: Remove deprecated muxpos (#20525)
+* cpu/sam0_common: avoid bitfield usage (#20747)
+* cpu/samd21: avoid the use of bitfield (#20713)
+* cpu/samd5x: avoid the use of bitfields (#20714)
+* cpu/samd5x: handle CAN errors (#20667)
+* cpu/saml1x: avoid the use of bitfield in register calls (#20710)
+* cpu/saml21: avoid the use of bitfield in register call (#20708)
+* cpu/stm32/periph/stm32_eth: provide confirm_send (#18428)
+* cpu/stm32: de-duplicate ifdefs (#20609)
+* periph/flashpage: remove deprecated flashpage_*_free functions (#18093)
+* pkg/lwip: make use of confirm send (#18427)
+* Remove extraneous bitwise assignment operation on NRF5X_common gpio.c (#20737)
+* sam0_eth: implement .confirm_send() to fix fragmented sending (#20666)
+* tree wide: compilation fixes for `native64` on musl systems (#20730)
+
+Device Drivers
+--------------
+
++ drivers/at24cxxx: Add M24C01 device and enhance documentation (#20588)
++ drivers/led: add LEDX_IS_PRESENT defines (#20637)
++ drivers/mtd: Add check for integer overflow (#20587)
++ Drivers: add MAX31855 thermocouple-to-digital converter (#20741)
+* drivers/at86rf215: port to new netdev API (#20672)
+* drivers/at: expose some internal API (#20702)
+* drivers/periph_wdt: fix auto starting of watchdog timer (#20755)
+
+Documentation
+-------------
+
++ boards/nucleo-c031c6: add pinout to doc (#20657)
++ boards/nucleo-l452re: add pinout to doc page (#20678)
++ boards/nucleo-l4r5zi and l496zg: add pinout to documentation page (#20707)
++ boards/nucleo144: add pinout diagrams to boards documentation (#20614)
++ doc/getting-started: add hint for correct access rights on serial
+  device (#20600)
++ doc/terminals: add putty and sort terminals (#20611)
+* /README.md: match doc (#20740)
+* boards/doc: Adafruit Feather nRF52840 Sense is popular (#20597)
+* boards/nucleo-f429zi & f439zi: documentation improvements (#20594)
+* boards/nucleo-f722ze: doc update (#20671)
+* boards/stm32l0538-disco: doc improvements (#20767)
+* boards: Update SAM0-based documentation (#20621)
+* doc/doxygen: remove default value for search form (#20681)
+
+Build System / Tooling
+----------------------
+
++ pkg: Add script to check third party package version (#20662)
++ tools/dhcpv6-pd_ia: fix: Add explicit subnet id (#20776)
+* dist/tools/buildsystem_sanity_check: make shellcheck happy (#20721)
+* dist/tools/tapsetup: make compatible with doas (#20712)
+* Makefile.include: place compile_commands.json by rule target (#20716)
+* makefiles/docker.inc.mk: Pin riotbuild version with BUILD_IN_DOCKER=1 (#20472)
```

Comment:
Maybe worth a highlighting note?

Reply:
Yes, that seems to be something users of docker are better aware of.